tools/prepare_scene_for_editing.py

```diff
@@ -11,10 +11,14 @@
 
     parser = argparse.ArgumentParser()
     parser.add_argument("--perforce_content_dir")
+    parser.add_argument("--unreal_project_dir", default=os.path.realpath(os.path.join(os.path.dirname(__file__), "..", "cpp", "unreal_projects", "SpearSim")))
```

Comment:
@RachithP what is the reason that `prepare_scene_for_editing` needs to be parameterized like this? We don't call it from `build_executable` or `build_paks`. Am I missing something?